### PR TITLE
Fix admins' "policy search" feature

### PR DESCRIPTION
**Pull request by forevermatt**
### Fixed
- Fix admins' "policy search" feature

### Changed
- Rename store holding query string data to urlQuery

### CODE REVIEW

File: src/data/query-string.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { params } from '@roxi/routify'
+import { route } from '@roxi/routify'
 import { derived } from 'svelte/store'
 
-export const query = derived(params, () => {
+export const urlQuery = derived(route, () => {
```
Comment (hobbitronics) [inline src/data/query-string.ts:4]: maybe I've forgotten the context but what is route doing here?

**Review by hobbitronics — APPROVED** [review]
Looks nice. Just one question.

File: src/data/query-string.ts (hunk 1 above)
Comment (forevermatt) [reply src/data/query-string.ts:4]: No, good question. That's the store that triggers changes to this derived store. Any time the Svelte `route` store changes (aka the URL changes), I want to recalculate what the query string parameters are. I happen to not use Svelte's route store for that calculation, but it's the event I want to listen for, so to speak.

Comment (hobbitronics) [reply src/data/query-string.ts:4]: ok, that makes sense.

**Pull request merged**